keypad/keypad.c

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "pico/stdlib.h"
+
+
+#define KEYPAD_NUM_ROWS           4
+#define KEYPAD_NUM_COLUMNS        4
+#define NO_KEY_PRESSED         '\0'  // Just a character that unlikely be
+                                     // present in keyboards, so it can by used
+                                     // to determine if no key is pressed.
+#define READ_MS_DELAY            10  // Delay between lectures to avoid noise
+
+
+const uint8_t keypad_rows_pins[KEYPAD_NUM_ROWS] = {
+    10,
+    11, 
+    12,
+    13,
+};
+
+
+const uint8_t keypad_columns_pins[KEYPAD_NUM_COLUMNS] = {
+    18,  // GP18
+    19,  // GP19
+    20,  // GP20
+    21,  // GP21
+};
+
+
+const char keypad_keys[KEYPAD_NUM_ROWS][KEYPAD_NUM_COLUMNS] = {
+    {'1', '2', '3', 'A'},
+    {'4', '5', '6', 'B'},
+    {'7', '8', '9', 'C'},
+    {'*', '0', '#', 'D'},
+};
+
+
+void pico_keypad_init(void){
```

Comment:
Small nitpick, but most of the code in this repo uses a space between `)` and `{` so it would be nice if you could match that style.

Reply:
Thanks for this feedback. Is there any kind of linter we can run to make sure the code we are pushing is following the expected format?

I fixed this as part of https://github.com/raspberrypi/pico-examples/pull/542/commits/b80d96d01cfe1e5804fcb42e81f534d3a2d783fb